checkout_sdk/checkout_response.py

```diff
@@ -0,0 +1,21 @@
+class ResponseWrapper(object):
```

Comment:
Skd support pytahon>3 so no need to inherit object.

Reply:
Nice catch. Will remove it.